content/courses/advanced-ada/chapters/expressions.rst

```diff
@@ -3,6 +3,391 @@ Expressions
 
 .. include:: ../../global.txt
 
+Expressions: Definition
+-----------------------
+
+According to the Ada Reference Manual, an expression "is a formula that defines
+the computation or retrieval of a value." Also, when an expression is evaluated,
+the computed or retrieved value always has an associated type.
```

Comment:
Consider "... an associated type known at compile-time."  
but that's a minor suggestion.

Reply:
Yes, I like it.